samples/charger_and_events/README.rst

```diff
@@ -0,0 +1,118 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER and EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The charger and events sample demonstrates how to control charger and receive events using npmx drivers.
```

Comment:
As that is the intro sentence, can we be more specific about the charger? Do you mean any specific charger, e.g., for any specific device or a type of device?